### PR TITLE
TF Keras image_test additional test cases covers

### PR DESCRIPTION
**Pull request by Dayananda-V**
1- array_to_img invlaid usage test case
2- img_transforms api test case

### CODE REVIEW

**Comment by Dayananda-V** [comment]
@fchollet 

Please help to review the test case

File: tensorflow/python/keras/preprocessing/image_test.py [hunk 1]
```diff
@@ -345,6 +345,27 @@ def test_img_utils(self):
     x = keras.preprocessing.image.img_to_array(img, data_format='channels_last')
     self.assertEqual(x.shape, (height, width, 1))
 
+    # Test invalid usage
+    # not 3D
+    with self.assertRaises(ValueError):
```
Comment (fchollet) [inline tensorflow/python/keras/preprocessing/image_test.py]: When testing `ValueError`, please check the error message against a regex pattern (see other examples in the codebase). This makes sure that you are catching the error you intend to be raised.

**Review by fchollet — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/preprocessing/image_test.py (hunk 1 above)
Comment (Dayananda-V) [reply tensorflow/python/keras/preprocessing/image_test.py]: review comments accepted and updated the changes, please review and thanks.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 22 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @fchollet, @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Review by pavithrasv — APPROVED** [review]

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by rthadur** [comment]
@Dayananda-V  can you please check build failures ?

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
